pyHS100/smartplug.py

```diff
@@ -95,29 +95,34 @@ def brightness(self) -> Optional[int]:
         return int(self.sys_info['brightness'])
 
     @brightness.setter
-    def brightness(self, value: int):
+    def brightness(self, state: Tuple[int, int]):
```

Comment:
Please do not overload the brightness setter, but add a separate brightness as instructed in my earlier comment.